src/Composite/CompositeParser.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Db\Pgsql\Composite;
+
+/**
+ * Composite type representation to PHP array parser for PostgreSQL Server.
+ */
+final class CompositeParser
+{
+    /**
+     * Converts composite type value from PostgreSQL to PHP array
+     *
+     * @param string $value String to convert.
+     */
+    public function parse(string $value): array|null
+    {
+        if ($value[0] !== '(') {
```

Comment:
When `$value` is empty string, this code throw warning.

Reply:
The parser is for real data retrieved from db. It seems there no cases when it might be empty string.